src/docs/product/issues/states-triage/index.mdx

```diff
@@ -0,0 +1,90 @@
+
+### Ignore
+
+Ignore an issue **permanently** (the default action when you press the button) if it's just noise and you'll likely never care about it. Or ignore it until a condition is met (at which point it becomes Unresolved and flagged for review). You can also set an issue to Ignore if you're working on the issue if you don't want to keep getting alerts.
+
+### Delete
+
+Delete an issue to remove it from the system. A new issue is created if a new instance of the issue (a new event) occurs.
+
+### Delete and Discard
+
+Delete and discard an issue to remove it from the system. A new issue is not created even if a new instance of the issue (a new event) occurs.
+
+### Reprocess
+
+[Reprocessing](/product/issues/reprocessing/) allows you to apply source maps and debug information files to error events that have already occurred.
+
+<Note>
+
+The Auto Resolve setting defines the amount of time an issue can go without being seen again (that is, no new events on that issue) before it's automatically resolved. To check if this has been defined for a project, go to **[Project] > Settings > General Settings** and check the Event Settings section.
+
+</Note>
+
+## Issue List Tabs
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+The Issues page is organized into tabs, each corresponding to a filtered list of issues:
+
+- All Unresolved (`is:unresolved`): All unresolved issues, including issues in the "For Review" tab.
+- For Review (`is:unresolved is:for_review`): Issues that need to be reviewed; for-review issues are a subset of all unresolved issues.
+- Ignored (`is:ignored`): All ignored issues.
+- Saved Searches: Select from a set of [premade](/product/sentry-basics/search/#premade-searches) and custom [saved searches](/product/sentry-basics/search/#organization-wide-saved-searches). The name of the tab changes based on your selection.
+
+These different lists help you with the triaging process.
+
+When any of the following conditions occur for an issue, that issue is displayed in the For-Review tab, or Review List:
+
+- A new issue is created; it's labeled "New Issue"
+- A resolved issue re-occurs; it's labeled "Regression"
+- An ignored issue meets its ignore conditions; it's labeled "Unignored"
+
+For-review issues are a subset of all unresolved issues.
+
+You can mark issues as reviewed, and [sentry.io](https://sentry.io) removes the reason label (the issue stays in the Unresolved state). Unlike ignoring an issue, this is a way to acknowledge an issue that you intend to fix at some point. This is only available on issues flagged for review.
```

Comment:
"to acknowledge that you've seen the update to the issue but don't want to ignore it, that is, you intend to fix it at some point in the future."

"this is only....." -> "this _action_ is only available" (for a bit more clarity)